spec/system/project_spec.rb

```diff
@@ -268,6 +268,50 @@
         expect(page).to have_content("New Project")
       end
     end
+
+    context "when the description is empty", :no_ci do
+      before do
+        @original_api_host = Rails.configuration.mediaflux["api_host"]
+        Rails.configuration.mediaflux["api_host"] = "0.0.0.0"
+        @session_id = sponsor_user.mediaflux_session
+      end
+
+      after do
+        Rails.configuration.mediaflux["api_host"] = @original_api_host
+      end
+
+      it "allows the projects to be created" do
```

Comment:
Same question... if description is required, then when the description is empty it should NOT allow the project to be required, right?